contracts/openzeppelin/token/ERC20/IERC20.sol

```diff
@@ -74,9 +74,5 @@ interface IERC20 {
      *
      * Emits a {Transfer} event.
      */
-    function transferFrom(
-        address from,
-        address to,
-        uint256 amount
-    ) external returns (bool);
+    function transferFrom(address from, address to, uint256 amount) external returns (bool);
```

Comment:
if possible, I would exclude `contracts/openzeppelin` folder from the formating as it is an imported library and it should be easily 1-1 comparable with the original one

Reply:
Added to lint/formatting ignores

Reply:
(And also reverted the formatting)